paprica/loader.py

```diff
@@ -46,7 +46,8 @@ def tile_from_apr(apr, parts):
                       neighbors=None,
                       neighbors_tot=None,
                       neighbors_path=None,
-                      frame_size=2048,
+                      frame_size_h=apr.shape(1),
+                      frame_size_v=apr.shape(2),
```

Comment:
given our weird convention of (z, x, y) corresponding to D, H, W, should these be swapped?

Reply:
or are the frames arrays of shape `(frame_size_h, frame_size_v)`?

Reply:
So python is (z, y, x) and that corresponds to (z, v, h) so I think v is indexed by 2 and h by 1, am I wrong?

Reply:
frames are (z, frame_size_v, frame_size_h), I used v for vertical and h for horizontal to avoid the ambiguity of (x, y, z)